examples/rust/robby_fischer/README.md

```diff
@@ -0,0 +1,317 @@
+First we create the recording and store it as the thread local recording in each thread.
+```rust
+
+let app_id = "RobbyFischer";
+let rec_id = uuid::Uuid::new_v4().to_string();
+let rec = rerun::RecordingStreamBuilder::new(app_id)
+    .recording_id(&rec_id)
+    .connect()
+    .unwrap();
+
+// …
+
+// Will be retrieved later using `rerun::RecordingStream::thread_local`
+RecordingStream::set_thread_local(rerun::StoreKind::Recording, Some(rec.clone()));
+
+// …
+
+// Thread that does all the image processing.
+let to_be_moved_rec = rec.clone();
+let _vision_handle = std::thread::spawn(move || {
+    RecordingStream::set_thread_local(rerun::StoreKind::Recording, Some(to_be_moved_rec));
+    // …
+});
+```
+
+### Robot arm
+First we will log the URDF using the official [URDF dataloader](https://github.com/rerun-io/rerun-loader-python-example-urdf).
+```rust
+// Rerun will find the dataloader in the `PATH` and use it to log `arm.urdf`.
+rec.log_file_from_path("arm.urdf", None, false).unwrap();
+
+// Sets the position of the arm and rotates it 180 degrees.
+rec.log(
+    "arm.urdf",
+    &rerun::Transform3D::from_translation_rotation(
+        [-0.185, 0.130, 0.04],
+        Rotation3D::AxisAngle(RotationAxisAngle::new([0., 0., 1.], Angle::Degrees(180.0))),
+    ),
+)
+```
+To log joint positions we must convert the joint positions to link transformations, we will do this with the help of the [k](https://docs.rs/k/latest/k/) crate which supports forward kinematics and is capable of loading URDF files.
+```rust
+
+let chain = k::Chain::<f32>::from_urdf_file(URDF_PATH).unwrap();
+
+// …
+
+chain.set_joint_positions(&positions).unwrap();
+chain.update_transforms();
+
+for link_name in chain.iter_links().map(|link| link.name.clone()) {
+    // …
+    // Extracts translation and rotation of `link_name` relative to it's parent and it's entity_path.
+    // …
+
+    rec.log(
+        entity_path,
+        &rerun::Transform3D::from_translation_rotation(
+            Vec3D::new(translation[0], translation[1], translation[2]),
+            Rotation3D::Quaternion(Quaternion(quat.coords.as_slice().try_into().unwrap())),
+        ),
+    ).unwrap();
+}
+```
+It's planned trajectory is visualized using [LineStrips3D](https://rerun.io/docs/reference/types/archetypes/line_strips3d).
+```rust
+let strip: Vec<Vec3> = // …
+rec.log(
+    "a8origin/trajectory",
+    &rerun::LineStrips3D::new([strip])
+        .with_radii([rerun::Radius::new_scene_units(0.002)]),
+).unwrap();
+
+// Move arm along trajectory …
+
+// Remove trajectory after we've moved along it.
+rec.log("a8origin/trajectory", &rerun::Clear::flat())
+    .unwrap();
+
+```
+
+### Board and pieces
+First it logs the position and meshes of the boards.
+```rust
+// The board is stored as a GLTF file so we use the `log_node` function from this example: https://github.com/rerun-io/rerun/tree/main/examples/rust/raw_mesh to log it.
+log_node(rec, "a8origin/board", self.board_scene.clone()).unwrap();
```

Comment:
we actually support GLTF directly as is, can't you log this as `Assets3D`?

Not blocking, we can fix this up later